fetch.js

```diff
@@ -0,0 +1,70 @@
+var Bluebird  = require('bluebird');
+var _defaults = require('lodash/object/defaults');
+
+var EmberCliDeployError = require('./errors/ember-cli-deploy-error');
+
+var redis = require('then-redis');
+var redisClient;
+var defaultConnectionInfo = {
+  host: "127.0.0.1",
+  port: 6379
+};
```

Comment:
I was trying to match what `then-redis` does, but perhaps it would be safer to require passing a config at all times, or at least warn if it's not there.